root/utility/voting.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+type PropsT = {
+  +election: AutoEditorElectionT,
+  +user?: EditorT,
+};
+
+export function isInvolved(election: AutoEditorElectionT, user: ?EditorT) {
+  return !!user && (
+    election.proposer.id === user.id ||
+    election.candidate.id === user.id ||
+    (election.seconder1 && election.seconder1.id === user.id) ||
+    (election.seconder2 && election.seconder2.id === user.id)
+  );
+};
```

Comment:
Added https://github.com/metabrainz/musicbrainz-server/commit/ea051f8c39fb08679f6d1cede75403107d8160d1 ;)

(There's no case where a semicolon is ever needed after a function declaration, only function expressions.)